Github Actions currently broken

```
Generating autoload files
PHP 7.3.15-3+ubuntu18.04.1+deb.sury.org+1 is installed.
Found ImageMagick: /usr/bin/convert. Image thumbnailing will be enabled if you enable uploads.
Found the Git version control software: /usr/bin/git.
Using server name "http://localhost".
Using server URL "http://localhost/w".
Warning: Your default directory for uploads (/home/runner/mediawiki/images/) is not checked for vulnerability to arbitrary script execution during the CLI install.
Using the PHP intl extension for Unicode normalization.
The environment has been checked. You can install MediaWiki.
Setting up database
Cannot access the database: :real_connect(): (HY000/2002): No such file or directory. Check the host, username and password and try again. If using "localhost" as the database host, try using "127.0.0.1" instead (or vice versa).
done
Cannot access the database: :real_connect(): (HY000/2002): No such file or directory. Check the host, username and password and try again. If using "localhost" as the database host, try using "127.0.0.1" instead (or vice versa).
##[error]Process completed with exit code 1.
```

--- a/.github/workflows/install-mediawiki.sh
+++ b/.github/workflows/install-mediawiki.sh
@@ -14,6 +14,7 @@ php "${HOME}/mediawiki/maintenance/install.php" \
   --dbname testwiki \
   --dbuser root \
   --dbpass root \
+  --dbport "${MYSQL_PORT}" \
   --scriptpath "/w" \
   testwiki admin
 echo -e "\n\nrequire_once __DIR__ . '/includes/DevelopmentSettings.php';" >> "${HOME}/mediawiki/LocalSettings.php"

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -34,20 +34,47 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    services:
+      mysql:
+        image: mysql:8
+        env:
+          MYSQL_USER: root
+          MYSQL_PASSWORD: root
+          MYSQL_DATABASE: testwiki
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306/tcp
+
     steps:
-    - run: sudo apt-get update && sudo apt-get install -y php-ast
     - uses: actions/checkout@v1
+
     - name: Setup PHP Action
       uses: shivammathur/setup-php@1.7.2
       with:
         php-version: '7.3'
-    - run: bash "${GITHUB_WORKSPACE}/.github/workflows/install-mediawiki.sh"
+
+    - name: Install PHP extension
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y php-ast
+
+    - name: Start the MySQL service
+      # See https://github.blog/changelog/2020-02-21-github-actions-breaking-change-ubuntu-virtual-environments-will-no-longer-start-the-mysql-service-automatically/
+      run: sudo /etc/init.d/mysql start
+
+    - name: Install MediaWiki
+      run: bash "${GITHUB_WORKSPACE}/.github/workflows/install-mediawiki.sh"
+      env:
+        MYSQL_PORT: ${{ job.services.mysql.ports[3306] }}
+
     - name: Install extensions
       run: |
         bash "${GITHUB_WORKSPACE}/.github/workflows/install-extension.sh" Flow
         bash "${GITHUB_WORKSPACE}/.github/workflows/install-extension.sh" Echo
         bash "${GITHUB_WORKSPACE}/.github/workflows/install-extension.sh" Renameuser
-    - run: php "${HOME}/mediawiki/extensions/Sanctions/vendor/bin/phan" -d "${HOME}/mediawiki/extensions/Sanctions" -p -m checkstyle
+
+    - name: Phan
+      run: php "${HOME}/mediawiki/extensions/Sanctions/vendor/bin/phan" -d "${HOME}/mediawiki/extensions/Sanctions" -p -m checkstyle
 
   phpunit:
     strategy:
@@ -59,11 +86,33 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    services:
+      mysql:
+        image: mysql:8
+        env:
+          MYSQL_USER: root
+          MYSQL_PASSWORD: root
+          MYSQL_DATABASE: testwiki
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+          - 3306/tcp
+
     steps:
     - uses: actions/checkout@v1
+
     - name: Setup PHP Action
       uses: shivammathur/setup-php@1.7.2
       with:
         php-version: '7.3'
-    - run: bash "${GITHUB_WORKSPACE}/.github/workflows/install-mediawiki.sh"
-    - run: php "${HOME}/mediawiki/tests/phpunit/phpunit.php"
+
+    - name: Start the MySQL service
+      # See https://github.blog/changelog/2020-02-21-github-actions-breaking-change-ubuntu-virtual-environments-will-no-longer-start-the-mysql-service-automatically/
+      run: sudo /etc/init.d/mysql start
+
+    - name: Install MediaWiki
+      run: bash "${GITHUB_WORKSPACE}/.github/workflows/install-mediawiki.sh"
+      env:
+        MYSQL_PORT: ${{ job.services.mysql.ports[3306] }}
+
+    - name: PHPUnit
+      run: php "${HOME}/mediawiki/tests/phpunit/phpunit.php"

